### PR TITLE
MCR-2236 use @XmlElementWrapper on derivate listing

### PR DESCRIPTION
**Pull request by yagee-de**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2236).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 57ca829daaf72fce2ed7b42564cadcb55094213c! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5697-pull_request-1003

File: mycore-restapi/src/main/java/org/mycore/restapi/converter/MCRMetaDefaultListXMLWriter.java [hunk 1]
```diff
@@ -58,12 +59,17 @@ public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotat
...
-            .filter(a -> MCRParams.class.isAssignableFrom(a.annotationType()))
-            .map(MCRParams.class::cast)
-            .flatMap(p -> Stream.of(p.values()))
-            .filter(p -> p.name().equals(PARAM_XMLWRAPPER))
+            .filter(a -> XmlElementWrapper.class.isAssignableFrom(a.annotationType()))
             .findAny()
-            .map(MCRParam::value);
+            .map(XmlElementWrapper.class::cast)
+            .map(XmlElementWrapper::name)
+            .or(() -> Stream.of(annotations)
```
Comment (rsteph-de) [inline mycore-restapi/src/main/java/org/mycore/restapi/converter/MCRMetaDefaultListXMLWriter.java]: I would not provide a fallback here, but cleanup the other usages instead - keep it simple !!!

**Review by rsteph-de — APPROVED** [review]
looks good for me

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 0dc2d8dc46e5f57d1fea5fb4023e9e34944f2824! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5706-pull_request-1003

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 0dc2d8dc46e5f57d1fea5fb4023e9e34944f2824! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5706-pull_request-1003

**Pull request merged**